pyquaticus/envs/pyquaticus.py

```diff
@@ -1440,6 +1494,9 @@ def state_to_obs(self, agent_id):
         obs["on_side"] = agent.on_own_side
         obs["tagging_cooldown"] = agent.tagging_cooldown
 
+        #Is tagged
+        obs["is_tagged"] = self.state["agent_tagged"][agent.id]
```

Comment:
If it's not too much trouble, could you actually create a new field in the agent's Player class? You would add a field here: https://github.com/mit-ll-trusted-autonomy/pyquaticus/blob/8c8ab6a5f330782569558e0d0f9f88d9b3ba71e3/pyquaticus/envs/pyquaticus.py#L144. Then make sure it's updated in the right place.